cpp/velox/compute/VeloxBackend.cc

```diff
@@ -113,6 +115,8 @@ void VeloxBackend::init(const std::unordered_map<std::string, std::string>& conf
   initJolFilesystem();
   initCache();
   initConnector();
+  velox::parquet::registerParquetReaderFactory();
+  velox::parquet::registerParquetWriterFactory();
```

Comment:
@JkSelf PTAL

Reply:
@zhouyuan Yes. We need to add this registration here due to this refactoring https://github.com/facebookincubator/velox/pull/8871.